# Encoder position [ticks]
float64 encoder_pos

# Encoder velocity [ticks/s]
float64 encoder_vel

# Driver current sense reading [Amps]
float64 current_sense
